EventFilter/SiPixelRawToDigi/plugins/SiPixelRawToDigi.cc

```diff
@@ -152,17 +153,15 @@ void SiPixelRawToDigi::produce(edm::Event& ev, const edm::EventSetup& es) {
   // initialize cabling map or update if necessary
   if (recordWatcher.check(es)) {
     // cabling map, which maps online address (fed->link->ROC->local pixel) to offline (DetId->global pixel)
-    edm::ESTransientHandle<SiPixelFedCablingMap> cablingMap;
-    es.get<SiPixelFedCablingMapRcd>().get(cablingMapLabel, cablingMap);  //Tav
+    edm::ESHandle<SiPixelFedCablingMap> cablingMap = es.getHandle(tCablingMap);
```

Comment:
NOTE: The previous iuse of `ESTransientHandle` here was inappropriate as the data product must not be allowed to be deleted before the end of the IOV since internal parts of the data product are being held.